### ISSUE TITLE
Issue with List<String>

### ISSUE THREAD

**Issue by dmitrylavrikov** [mcharmas/android-parcelable-intellij-plugin#32]
In release 0.6.1 generation for fields like `private List<String> names;` results in something like this:
```java
    @Override
    public void writeToParcel(Parcel dest, int flags) {
        dest.writeList(this.names);
    }

    private Contact(Parcel in) {
        this.names = new ArrayList<List<String>>();
        in.readList(this.names, List<String>.class.getClassLoader());
    }
```
Which, obviously, does not compile. :) 
I suggest that for lists of strings you should use `writeStringList()` and `readStringList()`, for example:
```java
    @Override
    public void writeToParcel(Parcel dest, int flags) {
        dest.writeStringList(this.names);
    }

    private Contact(Parcel in) {
        this.names = new ArrayList<String>();
        in.readStringList(this.names);
    }
```

**Comment by tomredman** [comment]
An excellent update!

**Comment by petersnoopy** [comment]
+1

**Comment by mcharmas** [comment]
This should be fixed in 0.6.2 (in review by JetBrains).
